projects/core/src/lib/layers/symbology/raster-symbology-editor/raster-symbology-editor.component.html

```diff
@@ -126,15 +126,19 @@
                 </ng-template>
             </mat-expansion-panel>
 
```

Comment:
![grafik](https://user-images.githubusercontent.com/4947061/221539802-f9284098-94c1-4dcf-89fa-6f962194b9f2.png)

The last button does not fill the space.

Reply:
Fixed this so it's displayed as rows again 7b9aacac815d6eac4398b0466d4c9b74d043a732

Reply:
Unfortunately, this fix did not lead to a better look and feel.

You should have the buttons side-by-side (cf. https://material.angular.io/components/button-toggle/overview) if the space is sufficient and it should span the whole row.
Only if the space is not enough, it should use more than one row.